### PR TITLE
Evaluation

### PR DESCRIPTION
**Pull request by joseguti84**
(Functionality in progress)
- Template: Name to each groupKey in the "GetEvaluationTemplateCommand.cs"
- Data mapping in viewModel
- Render all califications items dynamically from viewModel
- Improve the CSS styles

### CODE REVIEW

File: source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js [hunk 1]
```diff
@@ -13,25 +17,107 @@
         if (data) {
             this.fromJs(data);
         }
+        this.getAllCalificationsByKey = function (key) {
+            return _.map(this.califications(), function (calification) {
+                return calification.getCalificationByKey(key);
+            });
+        }
+        this.calificatorsHeaderTable = ko.computed(function () {
```
Comment (andresmoschini) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js]: Maybe it is simpler to keep `califications` only in the JSON Model and apply these mapping on parsing JSON, what do you think @joseguti84?

File: source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js [hunk 2]
```diff
@@ -94,30 +193,56 @@
 
     var Template = function (data) {
         this.items = ko.observableArray();
+        this.groups = ko.observableArray();
+        this.groupedItems = ko.computed(function () {
```
Comment (andresmoschini) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js]: The same here, I think that it is easier to understand if all this logic in on parsing data.

**Comment by AlphaGit** [comment]
:+1: 

I saw a few comments that could be removed but I know this is a WIP.

File: source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js (hunk 1 above)
Comment (joseguti84) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js]: I agree with you. I will change that

File: source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js (hunk 2 above)
Comment (joseguti84) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js]: Ok

**Comment by joseguti84** [comment]
Exactly, we will remove these comments in the next PR's

**Comment by joseguti84** [comment]
I closed this PR because we was re write the calification view model. So, I created a new PR

**Pull request closed without merging**